src/DynamoCore/Graph/Notes/NoteModel.cs

```diff
@@ -4,9 +4,18 @@
 
 namespace Dynamo.Graph.Notes
 {
+    /// <summary>
+    /// NoteModel represents notes in Dynamo.
+    /// Dynamo note is little square, where you can write something, 
```

Comment:
Remove all the lines from DynamoNote. Just say NoteModel represents notes in Dynamo